### DISCUSSION TITLE
FE: CSS framework to use

### DISCUSSION THREAD

**Discussion by mwakazi-dev** [DevSkillsHQ/discusssions#66] [Help] [answered]
Hello, for the css framework, Are we only limited to use tailwind?

**Comment by fenske** [comment, marked as answer]
Hi @jmwakz99 ! I'm Anton from Alva. Unless the hiring team explicitly mentioned it to you, feel free to use any other framework of choice.

Let us know if you have any other questions.

BR,
Anton.

**Reply by mwakazi-dev** [reply]
Hello @fenske I mean't the css framework?

**Reply by fenske** [reply]
That's right. Feel free to use any other lib apart from Tailwind (unless the hiring team specified it to you explicitly that they wanted the assignment to be completed using Tailwind).
